### PR TITLE
UE-2696: Make a simple wrapper for mesh compression

### PR DESCRIPTION
**Pull request by nbcong**
//cc @svensht2 
- These wrappers need for our using draco compression scheme(s) from visual studio 2013 projects.
- I think that we should keep this branch is the base branch for further work of draco integrations, so we do not need to merge it onto the master.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -763,6 +763,76 @@ else ()
...
+                  "${draco_src_root}/psy/psy_draco.h"
+                  "${draco_src_root}/psy/psy_draco_decoder.cpp"
+                  "${draco_src_root}/psy/psy_draco_decoder.h"
+                  "${draco_src_root}/psy/psy_draco_encoder.cpp"
+                  "${draco_src_root}/psy/psy_draco_encoder.h")
+      set_target_properties(psy_draco_compression PROPERTIES SOVERSION 1)
+  else()
+      # for consitency supporting on windows, we're also creating a wrapper for other platform
+      add_library(psy_draco_compression
+                  ${draco_attributes_sources}
```
Comment (svensht2) [inline CMakeLists.txt]: most of these sources are reused, can we avoid duplicated code?

**Comment by svensht2** [comment]
this is looking pretty good besides the comment above

File: CMakeLists.txt (hunk 1 above)
Comment (nbcong) [reply CMakeLists.txt]: addressed, thanks!

File: CMakeLists.txt [hunk 2]
```diff
@@ -763,6 +763,50 @@ else ()
...
+      ${draco_io_sources}
+      ${draco_mesh_sources}
+      ${draco_metadata_dec_sources}
+      ${draco_metadata_enc_sources}
+      ${draco_metadata_sources}
+      ${draco_point_cloud_sources}
+      ${draco_points_dec_sources}
+      ${draco_points_enc_sources}
+      ${draco_version_sources}
+      "${draco_src_root}/psy/psy_draco.h"
```
Comment (svensht2) [inline CMakeLists.txt]: maybe we should put these headers in include/psy/ directory?

**Comment by nbcong** [comment]
@svensht2 after the first integration version have been merged, we should close this PR and keep this branch as the base branch. So for further changes, we should make another branch and request merging to the base branch (just remind).

**Comment by svensht2** [comment]
Yep, agreed

**Pull request closed without merging**